hs_core/autocomplete_light_registry.py

```diff
@@ -37,7 +37,7 @@ class GroupAutocomplete(autocomplete_light.AutocompleteModelBase):
     search_fields=['name']
 
     def choices_for_request(self):
-        self.choices = self.choices.filter(gaccess__active=True)
+        self.choices = self.choices.filter(gaccess__active=True).exclude(name='Hydroshare Author')
```

Comment:
We need this fix system wide. The fact that this group is returned is an oversight.